### ISSUE TITLE
Feature: 'Double Bogey'

### ISSUE THREAD

**Issue by octaconta** [Layers-of-Railways/Railway#436]
### Describe the Suggestion

This mod lacks an intuitive way to make so called 'double' bogeys.
I want to create these bogeys to contribute to the mod.

They would be a bogey that is basically two bogeys in one that rotate around a central, vertical axis. This would allow more ease of use when trying to make, for example, a semi-truck using narrow-gauge rails underground. Since these trucks have a hitch which allows their trailer to rotate independently from (yet with) the vehicle itself, I think it would be useful to be able to create these in an intuitive fashion.

In short, a bogey that is split in two and that can rotate around a central axis while still being linked, but can also have blocks on that central axis.

### Additional Context

I have seen many people use very complex solutions to achieve this, so I feel it could be made easier.
